docs/application/web/guides/multimedia/metadata.md

```diff
@@ -23,27 +42,126 @@ Additionally, to access files using the Metadata API (in [mobile](../../api/late
+1. Create a file handle using `createFileHandle()`:
 
    ```javascript
    var filePath = "videos/sample_video.mp4";
    var fileHandle = tizen.metadata.createFileHandle(filePath);
    ```
 
-2. When the file handle is properly created, metadata can be extracted:
+2. When the file handle is created, metadata can be extracted:
 
    ```javascript
    console.log("Video duration is: " + fileHandle.get("DURATION") + " milliseconds");
    console.log("Video title is: " + fileHandle.get("TITLE"));
    ```
 
 3. For better performance, the file handle does not release underlying resources after every metadata extraction. The data is decoded on the first extraction, which helps in speeding up the next metadata extractions.
-When all the needed metadata is extracted, the file handle needs to be released using the `release()` method:
+When all the needed metadata is extracted, the file handle needs to be released using `release()`:
 
    ```javascript
    fileHandle.release();
    ```
 
+## Extract artwork
+
+1. Create a file handle using `createFileHandle()`. Ensure your file includes an artwork.
+
+   ```javascript
+   var filePath = "music/sample.mp3";
+   var fileHandle = tizen.metadata.createFileHandle(filePath);
+   ```
+
+2. When the file handle is created, artwork can be extracted:
+
+   ```javascript
+   var artwork = fileHandle.getArtwork();
+   ```
+
+3. Blob object can be easily shown in your application in html *&lt;img&gt;* tag
+
+   ```javascript
+   var elem = document.getElementById("blobImage");
+   elem.src = URL.createObjectURL(artwork);
+   ```
+
+4. After use, release the file handle.
+
+## Extract thumbnail frame
+
+1. Create a file handle using `createFileHandle()` for a video file.
+
+   ```javascript
+   var filePath = "videos/sample_video.mp4";
+   var fileHandle = tizen.metadata.createFileHandle(filePath);
+   ```
+
+2. When the file handle is created, thumbnail frame can be extracted:
+
+   ```javascript
+   var thumbnail = fileHandle.getThumbnailFrame();
+   ```
+
+3. Blob object can be easily shown in your application in html *&lt;img&gt;* tag
+
+   ```javascript
+   var elem = document.getElementById("blobImage");
+   elem.src = URL.createObjectURL(thumbnail);
+   ```
+
+4. After use, release the file handle.
+
+## Extract certain frame
+
+1. Create a file handle using `createFileHandle()` for a video file.
+
+   ```javascript
+   var filePath = "videos/sample_video.mp4";
+   var fileHandle = tizen.metadata.createFileHandle(filePath);
+   ```
+
+2. When the file handle is created, thumbnail frame can be extracted. You need to provide timestamp information (in milliseconds) and a flag indicating accuracy. When the flag value is set to 'true', flag indicates extracting an exact frame for the given time. When the value is 'false', flag indicates extracting an <a href="https://en.wikipedia.org/wiki/Video_compression_picture_types">I-frame</a> nearest to the given time. Gathering nearest I-frame has better performance.
```

Comment:
'true' 
'false'
Was the intention to use code tags? if yes please change to:


`true`  
`false`

Reply:
done